e2e/provision/install_sandbox.sh

```diff
@@ -66,22 +65,22 @@ rm -f ~/.ssh/id_rsa*
 echo -e "\n\n\n" | ssh-keygen -t rsa -N ""
 cat "$HOME/.ssh/id_rsa.pub" >>"$HOME/.ssh/authorized_keys"
 
-KVER=$(uname -r)
-
 if ! lsmod | grep -q gtp5g; then
-    [[ -d "$HOME/gtp5g" ]] || git clone --depth 1 -b v0.6.8 https://github.com/free5gc/gtp5g.git "$HOME/gtp5g"
-
-    pushd "$HOME/gtp5g" >/dev/null
-    GCC=gcc
-    if [[ $(uname -v) == *22.04.*-Ubuntu* ]]; then
-        GCC=gcc-12
+    if ! command -v gcc >/dev/null; then
```

Comment:
I had to look for 22.04, because I had a problem on that platform where they installed gcc didn't match the kernel headers, I had to specifically pick a version of gcc (gcc-12) that matches the kernel headers package.

Reply:
Ok, I think that we can limit the OS releases to Ubuntu focal and eventually cover others